app/views/comments/_single_comment.html.erb

```diff
@@ -14,7 +14,7 @@
     <% elsif single_comment.hidden_by_admin? && !logged_in_as_admin? %>
       <p class="message"><%= ts("(This comment is under review by an admin and is currently unavailable.)") %></p>
     <% else %>
-      <% cache single_comment do %>
+      <% cache(single_comment, :expires_in => 1.week ) do %>
```

Comment:
Er, but doesn't this mean that no edits on the comment will appear until it has expired one week later? That's definitely not going to go over well with users.